extensions/wikia/AdEngine3/src/wad/bt-loader.js

```diff
@@ -0,0 +1,112 @@
+import { utils } from '@wikia/ad-engine';
+
+const wikiaApiController = 'AdEngine3ApiController';
+const wikiaApiMethod = 'getRecCode';
+const isDebug = utils.queryString.get('bt-rec-debug') === '1';
+const placementClass = 'bt-uid-tg';
+const placementsMap = {
+		TOP_LEADERBOARD: {
+			uid: '5b33d3584c-188',
+			style: 'margin:10px 0; z-index:100;',
+			size: {
+				width: 728,
+				height: 90
+			},
+			lazy: false
+		},
+		TOP_BOXAD: {
+			uid: '5b2d1649b2-188',
+			style: 'margin-bottom:10px; z-index:100;',
+			size: {
+				width: 300,
+				height: 250
+			},
+			lazy: false
+		},
+		INCONTENT_BOXAD_1: {
+			uid: '5bbe13967e-188',
+			style: 'z-index:100;',
+			size: {
+				width: 300,
+				height: 250
+			},
+			lazy: true
+		},
+		BOTTOM_LEADERBOARD: {
+			uid: '5b8f13805d-188',
+			style: 'margin-bottom:23px; z-index:100;',
+			size: {
+				width: 728,
+				height: 90
+			},
+			lazy: true
+		}
+	};
+
+function duplicateSlot(slotName) {
+	const slot = document.getElementById(slotName);
+
+	if (slot) {
+		let node = document.createElement('span');
+
+		node.classList.add(placementClass);
```

Comment:
Just to make sure: we're ok with not logging these actions (adding class, setting data)? DOMElementTweaker did that, I don't know if there was a reason for it.

Reply:
Yeah, we're ok with that.